Tests/Admin/AdminHelperTest.php

```diff
@@ -110,27 +98,38 @@ public function testAddNewInstanceInflector()
         $object = $this->getMock('sdtClass', array('addEntry'));
         $object->expects($this->once())->method('addEntry');
 
-        $helper->addNewInstance($object, $fieldDescription);
+        $this->helper->addNewInstance($object, $fieldDescription);
     }
 
     public function testGetElementAccessPath()
     {
-        $container = $this->getMock('Symfony\Component\DependencyInjection\ContainerInterface');
+        $object = $this->getMock('stdClass', array('getPathToObject'));
+        $subObject = $this->getMock('stdClass', array('getAnother'));
+        $sub2Object = $this->getMock('stdClass', array('getMoreThings'));
 
-        $pool = new Pool($container, 'title', 'logo.png');
-        $helper = new AdminHelper($pool);
+        $object->expects($this->atLeastOnce())->method('getPathToObject')->will($this->returnValue(array($subObject)));
+        $subObject->expects($this->atLeastOnce())->method('getAnother')->will($this->returnValue($sub2Object));
+        $sub2Object->expects($this->atLeastOnce())->method('getMoreThings')->will($this->returnValue('Value'));
+
+        $path = $this->helper->getElementAccessPath('uniquePartOfId_path_to_object_0_another_more_things', $object);
+
+        $this->assertSame('path_to_object[0].another.more_things', $path);
+    }
 
+    /**
+     * @expectedException        Exception
+     * @expectdExceptionCode     0
+     * @expectedExceptionMessage Could not get element id from uniquePartOfId_path_to_object_0_more_calls Failing part: calls
```

Comment:
Please use the `setExpectedException` instead. It is better because you can't make mistake like line 121 (there is a typo)